desktop/src/apps/MailApp/MailApp.module.css

```
@@ -0,0 +1,1061 @@
+  text-align: right;
+  line-height: 1.5;
+  font-variant-numeric: tabular-nums;
+}
+
+.readBody {
+  padding: 22px 0 8px;
+  font-size: 14px;
+  line-height: 1.65;
+  color: var(--ml-text);
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)